sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -6,12 +6,17 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Compute</PackageId>
     <Description>Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
-    <Version>29.1.0</Version>
+    <Version>29.2.0</Version>
```

Comment:
Should I keep this version to 29.2.0 or change it back to 29.1.0.

Reply:
@hapandar I updated this one for minor version update in the past.  Is it not necessary?

Reply:
`29.2.0` is the correct version. A simple rule:
> Nuget Package version change (in .csproj), AssemblyFileVersion change. If it's a major version update, AssemblyVersion will also need to be changed (in AssemblyInfo.cs)